partial-update-tests/generated/src/main/java/fixtures/bodystring/StringOperationClient.java

```diff
@@ -14,26 +15,11 @@
 import com.azure.core.http.rest.RequestOptions;
 import com.azure.core.http.rest.Response;
 import com.azure.core.util.BinaryData;
-import fixtures.bodystring.implementation.EnumsImpl;
 import fixtures.bodystring.implementation.StringOperationsImpl;
 
 /** Initializes a new instance of the synchronous AutoRestSwaggerBatServiceClient type. */
 @ServiceClient(builder = AutoRestSwaggerBatServiceClientBuilder.class)
 public final class StringOperationClient {
-
-    // 1. manually add one field
-    private EnumsImpl enumClient;
-
-    /**
-     * 1. manually add one constructor
-     *
-     * @param serviceClient the async client.
-     * @param test the parameter.
-     */
-    StringOperationClient(StringOperationsImpl serviceClient, String test) {
-        this.serviceClient = serviceClient;
-    }
```

Comment:
Wondering why the change in PR would affect partial-update test (with does not involve customization class).

+ @haolingdong-msft